app/controllers/search_controller.rb

```diff
@@ -1,9 +1,18 @@
 class SearchController < ApplicationController
   before_action :require_user
   def index
-    conn = Faraday.get("https://damp-forest-93176.herokuapp.com/images/#{params[:search]}")
-    json = JSON.parse(conn.body, symbolize_names: true)
-    @image_options = ImageOptions.new(json)
+    search = params[:search].split.join('+')
+    if search.match('[a-zA-Z]+')
```

Comment:
This line confirms all characters are letters or spaces replaced by '+' for the sake of the API searches.